.github/workflows/start-release.yml

```diff
@@ -207,9 +225,46 @@ jobs:
             2>&1); then
 
             if grep "HTTP 422" <<< "$http_code"; then
-              echo "Milestone ${{needs.variables.outputs.milestone}} already exists." >> $GITHUB_STEP_SUMMARY
+              echo "::notice::Milestone ${{needs.variables.outputs.milestone}} already exists. Close it once it's finished."
             else
               echo "::error::Couldn't create milestone ${{needs.variables.outputs.milestone}}: $http_code"
               exit 1
             fi
+          else
+            echo "::notice::Milestone ${{needs.variables.outputs.milestone}} has been created. Close it once it's finished."
           fi
+
+  notify:
+    name: Notify everybody
+    needs: [variables, branch, milestone]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post to Slack
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        uses: slackapi/slack-github-action@v1.19.0
+        with:
+          channel-id: ${{env.slack_channel}}
+          payload: >-
+            { "blocks": [
+
+            { "type": "section", "text": { "type": "mrkdwn", "text":
+            ":white_check_mark: *Upstream release ${{needs.variables.outputs.release-patch}}
+            has been triggered on <${{github.server_url}}/${{github.repository}}|${{github.repository}}>.*" }},
+
+            { "type": "section", "text": { "type": "mrkdwn", "text":
+            "Branch `${{needs.variables.outputs.branch}}`
+            and milestone `${{needs.variables.outputs.milestone}}`
+            have been created.\n\nPRs merged to the ${{env.main_branch}} branch
+            and assigned to RC milestones will be cherry-picked when closing the respective milestones.
+            For the urgent fixes that must go exclusively to this release
+            open PRs to the `${{needs.variables.outputs.branch}}` branch." }},
```

Comment:
I think one opens a PR `against` something, not `to` something.

Reply:
My thinking was we request a pull to the branch. But my English is against me.

Reply:
Yes, we pull `to` the branch. I checked the entire form and there is `pull request against master`. Not sure how to explain it 😅